### PR TITLE
Feature/enclude protocols

### PR DESCRIPTION
**Pull request by tianxiuali**
# Description

Swap optimize

# Ticket

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/pages/InvestNew/Withdraw/index.jsx [hunk 1]
```diff
@@ -507,7 +520,7 @@ export default function Withdraw({
 
   useEffect(() => {
     if (isEmpty(address) || isEmpty(VAULT_ADDRESS) || isEmpty(VAULT_ABI)) return
-    const timer = setInterval(getPegTokenPrice(), 10000)
+    const timer = setInterval(getPegTokenPrice(), 1000000)
```
Comment (jerrylib) [inline src/pages/InvestNew/Withdraw/index.jsx:523]: 这个时间不能修改

File: src/pages/InvestNew/index.jsx [hunk 2]
```diff
@@ -181,12 +180,12 @@ function Invest(props) {
           }
         })
         .catch(noop)
-    const timer = setInterval(loadTotalAssetsFn, 3000)
+    const timer = setInterval(loadTotalAssetsFn, 300000)
```
Comment (jerrylib) [inline src/pages/InvestNew/index.jsx:183]: 这个时间不能修改

**Review by jerrylib — CHANGES_REQUESTED** [review]

File: src/pages/InvestNew/Withdraw/index.jsx (hunk 1 above)
Comment (tianxiuali) [reply src/pages/InvestNew/Withdraw/index.jsx]: 已修改

File: src/pages/InvestNew/index.jsx (hunk 2 above)
Comment (tianxiuali) [reply src/pages/InvestNew/index.jsx]: 已修改

**Review by jerrylib — APPROVED** [review]
OK

**Pull request merged**